client/src/App.js

```diff
@@ -1,13 +1,24 @@
-import React from 'react';
-import './App.css';
-import AuthForm from './components/Forms/AuthForm/AuthForm';
-
-function App() {
-  return (
-    <div className="App">
-      <AuthForm />
-    </div>
-  );
-}
-
-export default App;
+import React, { useState } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import AuthForm from 'components/Forms/AuthForm/AuthForm';
+import Navbar from 'components/Navbar/Navbar';
+import HomePage from 'pages/HomePage/HomePage';
+import ProfilePage from 'pages/ProfilePage/ProfilePage';
+import 'App.scss';
+
+const App = () => {
+  const [hasTokens, setHasTokens] = useState(true);
+
+  return (
+    <div className="App">
+      <AuthForm />
```

Comment:
should be rendered conditionally based on hasTokens

!hasTokens ? <AuthForm /> : .........